src/components/forms/PlanForm/index.tsx

```diff
@@ -209,6 +219,15 @@ const PlanForm = (props: PlanFormProps) => {
     if (values.interventionType === InterventionType.MDAPoint) {
       return mdaPointActivities;
     }
+    if (values.interventionType === InterventionType.DynamicFI) {
```

Comment:
can we code this as a map instead of a sequence of ifs?

Reply:
Changed this to use a map